wdio.conf.js

```diff
@@ -1,7 +1,7 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
-const defaultWdioConfig = require('terra-toolkit/config/wdio/wdio.conf');
+const { config } = require('@cerner/terra-functional-testing');
 
-const wdioConfig = defaultWdioConfig.config;
+const wdioConfig = config;
 
 const travis = process.env.TRAVIS;
```

Comment:
This travis stuff in the wdio config can be removed with corresponding changes to the travis.yml file.